### DISCUSSION TITLE
3.2.1rc0

### DISCUSSION THREAD

**Discussion by snowman2** [pyproj4/pyproj#950] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/pyproj4/pyproj/releases/tag/3.2.1rc0'>3.2.1rc0</a>.</em>

**Comment by snowman2** [comment]
The plan is to release with PROJ 8.1.1 in the wheels.

Release Candidate 0 Status:
- [x] Linux/OSX wheels (https://github.com/pyproj4/pyproj-wheels/pull/41)
- [ ] Windows wheels (@cgohlke)
- [x] Conda Forge (https://github.com/conda-forge/pyproj-feedstock/pull/104)
- [x] Debian (@sebastic)
- [x] Fedora (@jdkloe)

Plan to release sometime this weekend. Testing is always appreciated, but definitely not required.

**Comment by sebastic** [comment]
Debian package builds looking good so far:

 https://buildd.debian.org/status/package.php?p=python-pyproj&suite=experimental

**Reply by snowman2** [reply]
Great, thanks for testing 👍

**Comment by jdkloe** [comment]
Fedora building and testing also ran without issues.

**Reply by snowman2** [reply]
Perfect, thanks for testing 👍

**Comment by snowman2** [comment]
https://github.com/pyproj4/pyproj/releases/tag/3.2.1
Should be on pypi soon.
